### PR TITLE
[wrangler] update unenv

### PR DESCRIPTION
**Pull request by vicb**
Pin the version to unenv-nightly@2.0.0-1725621861.f1c0bb8

## What this PR solves / how to test

Fixes #[insert GH or internal issue number(s)].

## Author has addressed the following

- Tests
  - [x] Not necessary because: update unenv used by wrangler, covered by existing tests
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [x] Required / Maybe required
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [x] Included
- Public documentation
  - [x] Not necessary because: no functional change

### CODE REVIEW

**Review by RamIdeas — APPROVED** [review]

File: packages/wrangler/package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
 		"resolve.exports": "^2.0.2",
 		"selfsigned": "^2.0.1",
 		"source-map": "^0.6.1",
-		"unenv": "npm:unenv-nightly@2.0.0-1724863496.70db6f1",
+		"unenv": "npm:unenv-nightly@2.0.0-1725621861.f1c0bb8",
```
Comment (RamIdeas) [inline packages/wrangler/package.json:87]: https://github.com/cloudflare/workers-sdk/pull/6653/files#r1750586204

File: .changeset/ten-dancers-do.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Update unenv to the unenv-nightly@2.0.0-1725621861.f1c0bb8
```
Comment (RamIdeas) [inline .changeset/ten-dancers-do.md:5]: This needs updating to match the version update you just force pushed

File: packages/wrangler/package.json (hunk 1 above)
Comment (RamIdeas) [reply packages/wrangler/package.json]: thanks @vicb! the changeset also needs updating now too :)

File: .changeset/ten-dancers-do.md (hunk 2 above)
Comment (vicb) [reply .changeset/ten-dancers-do.md]: Good catch, thanks!

**Review by RamIdeas — APPROVED** [review]

**Review by petebacondarwin — APPROVED** [review]

**Comment by vicb** [comment]
@petebacondarwin @RamIdeas I've marked this as draft as I'd like to get one more change in `unenv` before merging this (https://github.com/unjs/unenv/pull/302). Sorry for that.

**Pull request closed without merging**